Blockzilla/AppDelegate.swift

```diff
@@ -18,6 +19,32 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             BuddyBuildSDK.setup()
         #endif
 
+        // Set up Telemetry
+        let telemetryConfig = Telemetry.default.configuration
+        telemetryConfig.appName = "Focus"
+        telemetryConfig.dataDirectory = .documentDirectory
```

Comment:
I don't really have a good idea for the dataDirectory but there are some hints on https://developer.apple.com/icloud/documentation/data-storage/index.html about where to store temporary files.